tests/sqllogictests/src/main.rs

```diff
@@ -142,7 +142,10 @@ async fn create_databend(client_type: &ClientType) -> Result<Databend> {
             client = Client::Clickhouse(ClickhouseHttpClient::create()?);
         }
     }
-    client.create_sandbox().await?;
+    let suites = SqlLogicTestArgs::parse().suites;
+    if suites != "tests/fuse-compat/compat-logictest" {
```

Comment:
To take care of compatibility testing, `set sandbox_tenant` was not supported in past versions.

Maybe we should find a better way to test compat.  Such as use the traditional test way not using sqllogictest.

cc @Xuanwo @sundy-li

Reply:
ok